src/components/atoms/icon/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Icon = ({ className, icon }) => <img className={className} src={icon} alt="close" />
+
+Icon.propTypes = {
+  icon: PropTypes.string.isRequired,
+  className: PropTypes.string,
+}
+
+Icon.defaultProps = {
+  className: undefined,
```

Comment:
I can be wrong, but isn't better to set the default to null instead of undefined? Because the undefined will be used as string on the className, different from null that will be not be set as element class.

Reply:
I just following the ``NTK`` project style guide but we can standardize